modules/experimental-layers/src/path-outline-layer/path-outline-layer.js

```diff
@@ -136,7 +136,10 @@ export default class PathOutlineLayer extends PathLayer {
     attribute.value = pathTesselator._updateAttribute({
       target: attribute.value,
       size: 1,
-      getValue: (object, index) => getZLevel(object, index) || 0
+      getValue: (object, index) => {
+        const zLevel = getZLevel(object, index);
+        return isNaN(zLevel) ? 0 : [zLevel];
```

Comment:
Why is this function sometimes returning a number sometimes returning an array?

Reply:
1. Your implementation should not rely on some niche behavior of another utility. The other function could be changed from checking `value.length` to Array.isArray(value)` and your layer will be broken.
2. If an accessor ever returns NaN, the layer should throw an error instead of silently not updating the attribute. It is certainly not the user's intention and they should know it.